pkg/commands/vcl/snippet/snippet_test.go

```diff
@@ -159,7 +156,6 @@ func TestVCLSnippetCreate(t *testing.T) {
 						Content:        i.Content,
 						Dynamic:        i.Dynamic,
 						Name:           i.Name,
-						Priority:       i.Priority,
```

Comment:
What's happening here?

Reply:
@triblondon The priority field wasn't used as part of the test. I thought no point in setting it in case it misled someone into thinking it was required for the test to pass.